src/components/login/Login.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Formik, Field, Form, ErrorMessage } from 'formik';
+import * as Yup from 'yup';
+import { LOGIN_USER } from '../../redux/actions/actionTypes';
+
+const initialValues = {
+    email: 'RamRanch@gmail.com',
+    password: '********' 
+}
+
+const LoginSchema = Yup.object().shape({
+    email: Yup.string()
+    .email("Invalid email"),
+    password: Yup.string()
+    .min(5, "Too Short!")
+    .max(50, "Too Long!")
+});
+
+const Login = () => {
+  return (
+      <Formik
+        initialValues = {initialValues}
+        validationSchema={LoginSchema}
+        render={({ errors, status, touched }) => (
+            <Form>
+            <div className="form-group">
+                <label htmlFor="email">Email</label>
+                <Field name="email" type="text" className={'form-control' + (errors.email && touched.email ? ' is-invalid' : '')} />
```

Comment:
is this className needed for form validation?

Reply:
Yeah, it gives the input field a red outline and it works with the Formik ErrorMessage component to display an error
![p1](https://user-images.githubusercontent.com/39246339/61596378-b5040e80-abd0-11e9-8521-2dbacb512179.PNG)